src/main/java/io/supertokens/featureflag/EE_FEATURES.java

```diff
@@ -17,7 +17,7 @@
 package io.supertokens.featureflag;
 
 public enum EE_FEATURES {
-    ACCOUNT_LINKING("account_linking"), MULTI_TENANCY("multi_tenancy"), TEST("test");
+    ACCOUNT_LINKING("account_linking"), MULTI_TENANCY("multi_tenancy"), TEST("test"), DASHBOARD_LOGIN("dashboard_login");
```

Comment:
```suggestion
    ACCOUNT_LINKING("account_linking"), MULTI_TENANCY("multi_tenancy"), TEST("test"), DASHBOARD_LOGIN("dashboard_emailpassword_login");
```

Same as the table name for consistency